sdks/python/tox.ini

```diff
@@ -319,6 +319,9 @@ deps =
   -r build-requirements.txt
   19: torch>=1.9.0,<1.10.0
   110: torch>=1.10.0,<1.11.0
+  111: torch>=1.11.0,<1.12.0
+  112: torch>=1.12.0,<1.13.0
+  113: torch>=1.13.0,<1.14.0
```

Comment:
In order for this to work, I think we need to:

1) Add more versions like this - https://github.com/apache/beam/blob/55d8ecb2584e711f8c382f71a7fca39b86a89e67/sdks/python/test-suites/tox/py38/build.gradle#L112
2) Update line 317 to be `[testenv:py{37,38,39,310}-pytorch-{19,110,111,112,113}]`

Right now, these tests aren't getting run as part of the precommit - https://ci-beam.apache.org/job/beam_PreCommit_Python_Commit/26635/consoleFull